specification/gedcom-03-datamodel.md

```diff
@@ -2100,6 +2100,13 @@ See also `FAMILY_EVENT_STRUCTURE`.
 
 An enumerated value from set `g7:enumset-MEDI` providing information about the media or the medium in which information is stored.
 
+When `MEDI` is a substructure of a `g7:CALN`, it is recommended that its payload describes the medium directly found at that call number rather than a medium from which it was derived.
```

Comment:
This is fine, but can we also add a note saying that `MEDI` as a substructure of a `g7:FORM` is not recommended to use and may go away in a future version of GEDCOM?